Rover/mode_manual.cpp

```diff
@@ -18,11 +18,13 @@ void ModeManual::update()
         rover.balancebot_pitch_control(desired_throttle);
     }
 
-    // walking robots support roll and pitch
-    float desired_roll, desired_pitch;
+    // walking robots support roll, pitch and walking_height
+    float desired_roll, desired_pitch ,desired_walking_height;
```

Comment:
tiny little white space issue here.  command should be right after "desired_pitch"

Reply:
fixed,thanks